src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/config/WebClientConfiguration.kt

```diff
@@ -43,17 +48,4 @@ class WebClientConfiguration(
   fun bankHolidayApiWebClient(): WebClient {
     return webClientBuilder.baseUrl(bankHolidayApiUrl).build()
   }
```

Comment:
There's something else here. I think the `authorizedClientManager` might also check the validity of the token received in the request - it has a URL to the oauth2 provider (auth) to get its public key to be able to do this. What you have it a direct pass-thru of received token out into the request headers, with no check that it is valid. This might be ok - as the prison API will also do this check, but it feels like the checking should still be done?

Reply:
Will speak to Pete about how the validating of tokens work

Reply:
Have spoken to Simon Mitchell from the tech team - he's had a look and is happy with the way it's been configured. He agreed that the authorizationManager isn't related to token verification. He will take this off as a separate task and discuss with the tech team to clarify exactly what happens. In the meantime we can press on.

He also recommended adding extra app-insight related config. Refer to https://github.com/ministryofjustice/hmpps-manage-users-api/tree/main/src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config
Add 
1. ClientTrackingConfiguration.kt
2. ApplicationInsightsConfiguration.kt
Raised ticket CRS-606